python/lsst/cp/pipe/ptc.py

```diff
@@ -356,8 +356,13 @@ def runDataRef(self, dataRefList):
             linearizer = linearityResults.outputLinearizer
 
             self.log.info("Writing linearizer:")
-            butler.put(linearizer, datasetType='Linearizer', dataId={'detector': detNum,
-                       'detectorName': detName, 'calibDate': calibDate})
+
+            detName = detector.getName()
+            now = datetime.datetime.utcnow()
+            calibDate = now.strftime("%Y-%m-%d")
```

Comment:
The `calibDate` is a required field for ingestion to work properly, but this will need to change in gen3.  There's a name duplication where `calibDate` is used as `validStart` in some places in the stack (calib ingest, for example), and as the calib creation date elsewhere.  This is on my list of things to clean up before declaring gen3 cpp "finished".  DMTN-148 will have a discussion of this issue as well.